sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -498,7 +498,7 @@ Azure::Response<Models::MergeEntityResult> TableClient::MergeEntity(
 {
   auto url = m_url;
   url.AppendPath(
-      m_tableName + PartitionKeyFragment
+      Azure::Core::Url::Encode(m_tableName) + PartitionKeyFragment
```

Comment:
Is there a functional difference between `Url::Encode(a+b+c)` and `Url::Encode(a)+Url::Encode(b)+Url::Encode(c)`? 

Naively, it seems like the first would be more efficient and easier to understand.

Reply:
done